Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:buster as base
+
+ENV PYTHONPATH=${PYTHONPATH}:${PWD} 
+RUN pip3 install poetry gunicorn markupsafe==2.0.1
+RUN mkdir /todo_app 
+COPY /todo_app /todo_app
+COPY pyproject.toml /
+COPY .env /
```

Comment:
More importantly, we definitely shouldn't be copying the `.env` file into our image - particularly later on for when we'll want to distribute the image to people, we want the secrets to remain outside the image. For us here, that will mean passing secrets through at run time (e.g. with `--env-file .env` as one of our docker run arguments) rather than baking the secrets in at build time)